### PR TITLE
Update Kubeflow Pipeline ECR image and Helm charts

### PR DESCRIPTION
**Pull request by jsitu777**
**Description of your changes:**
- update awsconfig and helm chart to use the newly built pipeline image (https://gallery.ecr.aws/kubeflow-on-aws/ml-pipeline/api-server)

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

### CODE REVIEW

File: awsconfigs/apps/pipeline-static/kustomization.yaml [hunk 1]
```diff
@@ -12,3 +12,4 @@ generatorOptions:
   disableNameSuffixHash: true
 patchesStrategicMerge:
 - ./s3/disable-default-secret.yaml
+- ./s3/service-account.yaml
```
Comment (surajkota) [inline awsconfigs/apps/pipeline-static/kustomization.yaml:15]: why this change is in this PR?

File: charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+2023/03/17 11:44:54 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml:1]: bug in the helmify?

File: charts/apps/kubeflow-pipelines/rds-only/templates/Deployment/ml-pipeline-viewer-crd-kubeflow-Deployment.yaml [hunk 3]
```diff
@@ -29,14 +29,13 @@ spec:
       - env:
         - name: NAMESPACE
           value: ''
-          valueFrom: null
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-only/templates/Deployment/ml-pipeline-viewer-crd-kubeflow-Deployment.yaml:32]: can you confirm this was a bug previously?

File: charts/apps/kubeflow-pipelines/rds-s3-static/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-2023/03/05 23:26:52 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
+2023/03/17 11:45:04 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3-static/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml:1]: bug

File: charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml [hunk 5]
```diff
@@ -4,19 +4,19 @@ data:
...
   appName: pipeline
-  appVersion: 2.0.0-alpha.6
+  appVersion: 2.0.0-alpha.7
   autoUpdatePipelineDefaultVersion: 'true'
   bucketName: '{{ .Values.s3.bucketName }}'
   cacheDb: cachedb
   cacheImage: gcr.io/google-containers/busybox
   cacheNodeRestrictions: 'false'
   cronScheduleTimezone: UTC
-  dbHost: testa.c6uzclpednto.ca-central-1.rds.amazonaws.com
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml:14]: why didnt the unit test catch this?

File: charts/apps/kubeflow-pipelines/rds-s3-static/values.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 rds:
   dbHost: 
-  mlmdDb: metadb
+  mlmdDb:
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3-static/values.yaml:3]: revert?

File: charts/apps/kubeflow-pipelines/rds-s3/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml [hunk 7]
```diff
@@ -1,3 +1,4 @@
+2023/03/17 11:44:51 well-defined vars that were never replaced: kfp-app-version,kfp-app-name
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml:1]: maybe this bug is there at other components as well, lets find out

File: charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml [hunk 8]
```diff
@@ -245,11 +245,10 @@ data:
...
 kind: ConfigMap
 metadata:
-  annotations: {}
   labels:
     app: kubeflow-pipelines-profile-controller
     app.kubernetes.io/component: ml-pipeline
     app.kubernetes.io/name: kubeflow-pipelines
     application-crd-id: kubeflow-pipelines
-  name: kubeflow-pipelines-profile-controller-code-dhffkh7f55
+  name: kubeflow-pipelines-profile-controller-code
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml:253]: how come the suffix is getting removed here, but not in `charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-env-5252m69c4c-kubeflow-ConfigMap.yaml`

File: charts/apps/kubeflow-pipelines/rds-s3/values.yaml [hunk 9]
```diff
@@ -1,9 +1,9 @@
 rds:
   dbHost: 
-  mlmdDb:
+  mlmdDb:
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/rds-s3/values.yaml:3]: why these changes now?

File: charts/apps/kubeflow-pipelines/s3-only-static/values.yaml [hunk 10]
```diff
@@ -1,4 +1,4 @@
 s3:
   bucketName: 
   minioServiceHost: s3.amazonaws.com
-  minioServiceRegion:
+  minioServiceRegion:
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/s3-only-static/values.yaml:4]: revert?

**Review by surajkota — CHANGES_REQUESTED** [review]
We also need to check the latest [sync.py script](https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/apps/pipeline/s3/sync.py) from RC2 branch manifests and update that 

Take a look at this PR - https://github.com/awslabs/kubeflow-manifests/pull/456 and see what changed in awsconfigs/apps/pipeline/s3/sync.py file from upstream https://github.com/kubeflow/manifests/blob/v1.6.1/apps/pipeline/upstream/base/installs/multi-user/pipelines-profile-controller/sync.py

File: charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml (hunk 5 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml:14]: this was from some of Ryan's previous IRSA chart, he accidentally left hard-coded values

File: charts/apps/kubeflow-pipelines/rds-s3-static/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml (hunk 4 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-s3-static/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml:1]: this was not a bug by helmify. kustomize v5.0.0 automatically generate these warning message at the header of the yaml file

File: charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml (hunk 2 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml]: mannually removed. This will resolve after we use kustomize v5.0.1

File: charts/apps/kubeflow-pipelines/rds-only/templates/Deployment/ml-pipeline-viewer-crd-kubeflow-Deployment.yaml (hunk 3 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-only/templates/Deployment/ml-pipeline-viewer-crd-kubeflow-Deployment.yaml:32]: kustomize version issue. doesn't affect behavior, it;s null

File: charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml (hunk 2 above)
Comment (surajkota) [reply charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml]: please use 5.0.1 because there are multiple files where this is happening

File: charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml (hunk 5 above)
Comment (surajkota) [reply charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/pipeline-install-config-kubeflow-ConfigMap.yaml:14]: sure, shouldnt the helmify unit test fail after generation?

File: charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml (hunk 2 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml]: yea. All were fixed.

File: charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml (hunk 8 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml:253]: https://github.com/kubernetes-sigs/kustomize/issues/5047 again seems an issue from kustomize version

File: charts/apps/kubeflow-pipelines/rds-s3-static/values.yaml (hunk 6 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-s3-static/values.yaml:3]: it should be what ever specified in here: https://github.com/awslabs/kubeflow-manifests/blob/main/tools/helmify/template/kubeflow-pipelines/values/rds-s3-static/values.yaml I think it was manually changed during testing from previous PR and did not change back. What I have in this PR should be right.

File: charts/apps/kubeflow-pipelines/s3-only-static/values.yaml (hunk 10 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/s3-only-static/values.yaml:4]: It should be whatever describe from here when running helmify https://github.com/jsitu777/kubeflow-manifests/blob/main/tools/helmify/template/kubeflow-pipelines/values/s3-only-static/values.yaml

File: charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml (hunk 8 above)
Comment (surajkota) [reply charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml:253]: in that case, can you check why is it happening only in rds-s3 and not in vanilla or rds-only

File: charts/apps/kubeflow-pipelines/s3-only-static/values.yaml (hunk 10 above)
Comment (surajkota) [reply charts/apps/kubeflow-pipelines/s3-only-static/values.yaml:4]: got it, can we remove whitespaces from the templates?

Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/s3-only-static/values.yaml]: removed

File: charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml (hunk 8 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml:253]: you are right, both rds-only and vanilla have the hash-suffix

**Review by ryansteakley — APPROVED** [review]
